db/migrate/20210616133601_create_report_keywords.rb

```diff
@@ -0,0 +1,15 @@
+# rubocop:disable Rails/CreateTableWithTimestamps
+class CreateReportKeywords < ActiveRecord::Migration[6.0]
+  def change
+    create_table :report_keywords do |t|
+      t.string :name, null: false, index: { unique: true }
```

Comment:
While I think that uniqueness test in Rails is not useful, I would keep it here. Here it can be useful, it can speed up searching in some cases. The cost is not that high as with Rails.